### PR TITLE
Ensure live after reboot

### PR DESCRIPTION
**Pull request by jyejare**

### CODE REVIEW

File: upgrade/helpers/tools.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ def host_ssh_availability_check(host, timeout=7):
     :param int timeout: The polling timeout in minutes.
 
     """
-    _, ip = host_pings(host, timeout=1, ip_addr=True)
+    _, ip = host_pings(host, timeout=timeout, ip_addr=True)
```
Comment (jyejare) [inline upgrade/helpers/tools.py]: We cannot have timeout 1, in case satellite is installed on bare metal and personal satellite.

**Comment by jyejare** [comment]
DO NOT MERGE this as I cannot test this on 6.2 to 6.3 ... .upgrade is broken due to BZ - https://bugzilla.redhat.com/show_bug.cgi?id=1499402

**Comment by sghai** [comment]
Changes looks good.

**Comment by jyejare** [comment]
To verify it works:
Check Standalone Upgrade jobs 784th build.

**Comment by sghai** [comment]
ACK

**Pull request merged**